### ISSUE TITLE
calender behaves like beeing dead, ownCloud 9.1.0 pre alpha (git)

### ISSUE THREAD

**Issue by mmattel** [owncloud/calendar#283]
# 
### Steps to reproduce
1. updated core master to commit 0487b54
2. enable calendar
3. click at apps to calendar to start it
### Expected behaviour

The calendar should be usable
### Actual behaviour

calendar is totally dead, can only go back with the browser.
![image](https://cloud.githubusercontent.com/assets/3321281/13578589/2c2781b6-e499-11e5-9e20-1b0320bd5736.png)
### Server configuration

**Operating system**: Ubuntu 14.04

**Web server:** nginx

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version:** (see ownCloud admin page) ownCloud 9.1.0 pre alpha (git)

**Updated from an older ownCloud or fresh install:** update

**Where did you install ownCloud from:** via ocdev

**Calendar version:** (see ownCloud apps page) 0.82

**Signing status (ownCloud 9.0 and above):**

```
Integrity checker has been disabled. Integrity cannot be verified.
```

**The content of config/config.php:**

```
    "system": {
        "instanceid": "ocnyg5ougn5r",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "xxxx"
        ],
        "datadirectory": "\/var\/www\/xxxx\/data",
        "overwrite.cli.url": "https:\/\/xxxx",
        "dbtype": "mysql",
        "version": "9.1.0.0",
        "dbname": "oclg",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "Europe\/Vienna",
        "installed": true,
        "filelocking.enabled": "true",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379,
            "timeout": 0,
            "dbindex": 0
        },
        "assetdirectory": "\/var\/www\/xxxx",
        "asset-pipeline.enabled": true,
        "filesystem_check_changes": 1,
        "preview_max_scale_factor": 1,
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\Illustrator",
            "OC\\Preview\\Postscript",
            "OC\\Preview\\Photoshop",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\Raw"
        ],
        "loglevel": 1,
        "maintenance": false,
        "theme": ""
    }
}
```

**List of activated apps:**

```
  - activity: 2.3.0
  - comments: 0.3.0
  - dav: 0.2.1
  - federatedfilesharing: 0.2.0
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.6.0
  - files_sharing: 0.10.0
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - gallery: 15.0.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - updatenotification: 0.2.0
```
### Client configuration

**Browser:** Opera, IE11, Chrome

**Operating system:** W7x64
#### Web server error log

```
no corresponding log entry
```
#### ownCloud log (data/owncloud.log)

```
no corresponding log entry
```

**Comment by georgehrke** [comment]
> updated master to commit 0487b54

There is no such commit in the calendar

**Comment by georgehrke** [comment]
and please post your config.php as requested in the template ...
There is a reason it's in there ;)

**Comment by georgehrke** [comment]
Furthermore Opera is not supported.I won't fix this if it only appears in Opera

**Comment by mmattel** [comment]
@georgehrke 
sorry, I meant core master.
Tried with IE 11, same behaviour.
I updated the issue with the missing data.

**Comment by georgehrke** [comment]
>  "asset-pipeline.enabled": true,

This option is not supported, master of the calendar shows an error message when this option is enabled

> Tried with IE 11, same behavior.

Internet Explorer is not supported either ^^
We aim to support Firefox, Safari and Chrome.

**Comment by georgehrke** [comment]
dup of https://github.com/owncloud/calendar/issues/205

**Comment by mmattel** [comment]
Chrome doesn´t work either (updated above).
Note: Opear always worked.
There is as described no error message logged.
The old calendar worked btw having `"asset-pipeline.enabled": true` set.
The documentation does not say anything about having this disabled for using the calendar.
IMHO a no go, makes the calendar unusable for oC 9.0
Not personal, but disapointing...

**Comment by georgehrke** [comment]
this is an upstream issue, not something we can fix easily

**Comment by georgehrke** [comment]
> Not personal, but disapointing...

If you would have tested the calendar earlier, we might have been able to discover this earlier than just one day before the release ...
As a developer I can't test every single option from core and there is a reason why we have pre release versions

**Comment by mmattel** [comment]
Hopefully you find a way.
Just to say, but merged PR #268 did not show up anything on my system. Nor on the browser or in the logs. I would have recognized that.
Having asset-pipelining disabled, makes the calendar working. Chrome, Opera, IE11.
Using asset-pipelining improves performance greatly, that´s what it has been built for.
I remember once a issue with the updater app with the same background, https://github.com/owncloud/updater/issues/184. Maybe worth taking a look at.
Regarding testing: I do some coding and testing in core. I am not a professional programmer and my business life takes a lot of time. The rest is shared...
